### DISCUSSION TITLE
Push was rejected by remote, reason branch is currently checked out.

### DISCUSSION THREAD

**Discussion by frrobert2** [zeapo/Android-Password-Store#1447] [Q&A] [answered]
I have installed pass on my desktop with the git option and installed password-store on my phone and tablet.  Both Android devices can ssh into my desktop.  Both devices can pull from the desktop.  I added a password to password-store on my tablet and tried to do a push to the desktop and I get the following error "Push was rejected by remote, reason branch is currently checked out."

I figured I did not set something up correctly.  Any suggestions would be appreciated.

Thank you.

**Comment by msfjarvis** [comment, marked as answer]
The instructions [here](https://github.com/android-password-store/Android-Password-Store/wiki/Setting-up-Password-Store-to-sync-with-a-local-computer) mention a bare repository that needs to be used as a trampoline between your phone and the main `pass` repository. That will resolve the problem.

**Reply by frrobert2** [reply]
Thanks
